.classpath

```diff
@@ -70,5 +69,7 @@
 	<classpathentry kind="lib" path="testlib/simple-5.0.2.jar"/>
 	<classpathentry kind="lib" path="lib/zap-2.4.1.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="lib/metadata-extractor-2.8.1.jar"/>
+	<classpathentry kind="lib" path="lib/xmpcore-5.1.2.jar"/>
```

Comment:
The new libraries must be added to ```lib``` directory of the add-on (and the old one removed).